src/main/kotlin/no/nav/personbruker/dittnav/api/authentication/AuthenticatedUser.kt

```diff
@@ -1,16 +1,31 @@
-package no.nav.personbruker.dittnav.api.common
+package no.nav.personbruker.dittnav.api.authentication
 
-import java.util.*
+import java.time.Instant
 
-enum class IdentityClaim(val claimName: String) {
+
+data class AuthenticatedUser (
+    val ident: String,
+    val loginLevel: Int,
+    val token: String,
+) {
+
+    fun createAuthenticationHeader(): String {
+        return "Bearer $token"
+    }
+    override fun toString(): String {
+        return "AuthenticatedUser(ident='***', loginLevel=$loginLevel, token='***')"
+    }
+}
+
+enum class IdentityClaim(val claimName : String) {
 
     SUBJECT("sub"),
```

Comment:
Skal ikke være nødvendig med sub lenger

Reply:
Så vi trenger egentlig bare psd? Isåfall burde vel hele IdentityClaim klassen være unødvendig?